projects/goharbor/harbor/docker/linux/harbor-redis/Dockerfile

```diff
@@ -0,0 +1,20 @@
+ARG BASE_IMAGE # https://gallery.ecr.aws/eks-distro-build-tooling/eks-distro-base
```

Comment:
Why are we building this redis image when we have one? Same question for nginx and "harbor-db". I'm not sure fi there are any harbor specific changes to those containers